src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -75,8 +75,12 @@ def load_arguments(self, _):
         c.argument('load_balancer_outbound_ip_prefixes', type=str, validator=validate_load_balancer_outbound_ip_prefixes)
         c.argument('load_balancer_outbound_ports', type=int, validator=validate_load_balancer_outbound_ports)
         c.argument('load_balancer_idle_timeout', type=int, validator=validate_load_balancer_idle_timeout)
+        c.argument('nat_gateway_managed_outbound_ip_count', type=int)
```

Comment:
Could you please also add a validator for this option, since the help messages says it should be in the range [1, 16]?

Reply:
Sure.